runtime/lua/vim/treesitter.lua

```diff
@@ -119,10 +121,10 @@ end
 
 --- Determines whether a node is the ancestor of another
 ---
----@param dest table Possible ancestor
----@param source table Possible descendant node
+---@param dest userdata Possible ancestor |tsnode|
+---@param source userdata Possible descendant |tsnode|
 ---
----@returns (boolean) True if dest is an ancestor of source
+---@return boolean True if dest is an ancestor of source
```

Comment:
Does the `@p`annotations work ? That would make it easier to read in the final docs.
```suggestion
---@return boolean True if @p dest is an ancestor of @p source
```

Reply:
They do; I've learned of them too late. Will try to add it consistently.